wireless/bluetooth/bt_hcicore.c

```diff
@@ -1575,6 +1593,53 @@ int bt_driver_register(FAR struct bt_driver_s *btdev)
 
 void bt_driver_unregister(FAR struct bt_driver_s *btdev)
 {
+  int ret;
+  FAR struct bt_buf_s *buf;
+
+  /* Tell the tx thread that the module is disconnected.
+   */
+
+  g_btdev.tx_status = ENOTCONN;
+
+  /* Make sure the thread is not blocked by the semaphore.
+   */
+
+  nxsem_post(&g_btdev.ncmd_sem);
+
+  /* We create and push a packet into the tx queue to unblock the thread.
+   */
+
+  buf = bt_hci_cmd_create(0xf00f, 0);
```

Comment:
What 0xf00f means? It is better to use a defined symbol instead of a hard-code value

Reply:
Nothing. Any packet will do but I will change it to a reset opcode. The tx thread will not transmit at this point.